.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "db-snapshot"]
 	path = db-snapshot
 	url = ../db-snapshot
+[submodule "third_party/openmrs-module-sync"]
+	path = third_party/openmrs-module-sync
+	url = git@github.com:openmrs/openmrs-module-sync.git
```

Comment:
Ok, I'm going to copy it into the source tree instead. We'd have to change our build scripts etc to make this work, I'm reluctant to do that right now.